### PR TITLE
Added verification section to concepts page

### PR DESCRIPTION
**Pull request by tom-halpin**
## Hey, I just made a Pull Request!

Added tool verification section the concepts documentation page. Slight overlap in content with some of the running Backstage locally content. But having struggled to get Backstage running locally on Windows I think the concepts page is useful and hopefully this adds to it.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [X] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/getting-started/concepts.md [hunk 1]
```diff
@@ -19,3 +19,57 @@ configuring, and operating Backstage easier.
...
+
+```yarn```
+
+```bash
+yarn --version
+```
+
+Note: running the following command should display the version number and patch yarn if needed
+
+```bash
```
Comment (benjdlambert) [inline docs/getting-started/concepts.md:63]: This command will actually run `yarn install` maybe it's worth removing from this guide.

File: docs/getting-started/concepts.md [hunk 2]
```diff
@@ -19,3 +19,57 @@ configuring, and operating Backstage easier.
...
+```
+
+### Contribute to Backstage
+
+If you want to contribute to Backstage you should also ensure the following command runs as expected.
+
+```TypeScript```
+
+```bash
+tcs -version
```
Comment (benjdlambert) [inline docs/getting-started/concepts.md:74]: think this should be `tsc` and I think `yarn tsc` is probably better as we want to use the `typescript` version in the project.

**Comment by benjdlambert** [comment]
Hey :wave:

Looks like this is missing DCO and need to run prettier on the markdown doc to get this to pass 👍

**Comment by tom-halpin** [comment]
Thanks for the feedback @benjdlambert, appreciate it, I will take a look shortly

**Comment by freben** [comment]
Just a small friendly ping :)

**Pull request closed without merging**